api/src/main/java/io/strimzi/api/kafka/Crds.java

```diff
@@ -24,6 +24,7 @@
 import io.strimzi.api.kafka.model.KafkaRebalance;
 import io.strimzi.api.kafka.model.KafkaTopic;
 import io.strimzi.api.kafka.model.KafkaUser;
+import io.strimzi.api.kafka.model.nodepool.KafkaNodePool;
```

Comment:
I don't have a strong opinion, but I am not sure we really need this `nodepoll` layer under `model`.
Making a comparison and thinking about `KafkaRebalance` resource, we don't have it in a `rebalance` layer under `model` for example.

Reply:
TBH, I think the `io.strimzi.api.kafka.model` package is already loaded with all kinds of classes, often used only once. I think better separation into smaller packages is useful for things that are not shared in any way I would ideally love to split the existing classes into packages as well, but that would break the compatibility for anyone using the API. But from my point of view, this is a good way to go - so unless you have really strong feelings about this, I would prefer to keep it.

Reply:
I totally agree with refactoring and moving the others to smaller packages, if that was the rationale behind this choice then +1 from me.
I can totally understand your concern about breaking compatibility but I think we should move in that direction asap.

Reply:
Well, ok ... I can open an issue to refactor the existing as well. But I do not think that should be done in this PR anyway.

Reply:
> But I do not think that should be done in this PR anyway.

Absolutely agree.

Reply:
I opened #8628 for it.